### PR TITLE
Pipeline group edit page in new pipeline config SPA.

### PR DESCRIPTION
**Pull request by vrushaliwaykole**
Issue: #

Description:

### CODE REVIEW

**Comment by vrushaliwaykole** [comment]
Screenshot 2019-11-15 at 5 56 41 PM

Screenshot 2019-11-15 at 5 57 17 PM

**Comment by vrushaliwaykole** [comment]
`Name` field is disabled 

Screenshot 2019-11-25 at 11 13 14 AM

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/admin_pipelines/edit_pipeline_group_modal.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import {CollapsiblePanel} from "../../components/collapsible_panel";
+import {FlashMessage, FlashMessageModel, MessageType} from "../../components/flash_message";
+import {AutocompleteField, SuggestionProvider} from "../../components/forms/autocomplete";
+import {CheckboxField, TextField} from "../../components/forms/input_fields";
+import {Modal, Size} from "../../components/modal";
+import {Table} from "../../components/table";
+import styles from "../secret_configs/index.scss";
+
+class UsersProvider extends SuggestionProvider {
+  private readonly usersCache: string[] | undefined;
```
Comment (ketan) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/admin_pipelines/edit_pipeline_group_modal.tsx]: Seems like `undefined` cannot be passed in from the constructor.

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/admin_pipelines/edit_pipeline_group_modal.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+  getData(): Promise<Awesomplete.Suggestion[]> {
+    return new Promise<Awesomplete.Suggestion[]>((resolve, reject) => {
+      resolve(this.usersCache);
+    });
+  }
+}
+
+class RolesProvider extends SuggestionProvider {
+  private readonly rolesCache: string[] | undefined;
```
Comment (ketan) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/admin_pipelines/edit_pipeline_group_modal.tsx]: Seems like `undefined` cannot be passed in from the constructor.

**Pull request merged**